src/views/Characters/CharacterDetails.test.jsx

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+
+import CharacterDetails from './CharacterDetails';
+-describe('Character Details', () => {
+  it('Should render one character by id', async () => {
+    render(
+      <MemoryRouter>
+        <CharacterDetails initialEntries={['/characters/:id']} />
+      </MemoryRouter>
```

Comment:
For our initial entries, we will actually want to pass in an actual id, instead of /:id to render that character with the specific id. Then we will test for that character to be on the page. We will also want to render app within our memory router.